gvr-360video/app/src/main/java/org/gearvrf/gvr360video/Minimal360Video.java

```diff
@@ -33,7 +33,7 @@
     /** Called when the activity is first created. */
     @Override
     public void onInit(GVRContext gvrContext) {
-        GVRScene scene = gvrContext.getNextMainScene();
+        GVRScene scene = gvrContext.getMainScene();
```

Comment:
There might be a little confusion about the use of getMainScene(). Say I need to create an app with multiple scenes. 

With this model I would need to request the framework for the first scene and then create the next 2 scenes and then set them using setMainScene(). 

The call to getMainScene() in this case is just a special case where the framework creates the first scene for you and presents it to you the first time (making it more than a function that returns the current main scene). In this case, the user need to be educated not to create the first main scene but instead to request it from the framework.

I feel a cleaner approach would be if the user created all three scenes and set them one by one. 

Let me know what you think.